.eslintrc.js

```diff
@@ -2,29 +2,398 @@ module.exports = {
+        '@typescript-eslint/method-signature-style': 'off',
+        '@typescript-eslint/naming-convention': 'off',
+        '@typescript-eslint/no-array-constructor': 'off',
+        '@typescript-eslint/no-base-to-string': 'off',
+        '@typescript-eslint/no-confusing-non-null-assertion': 'off',
+        '@typescript-eslint/no-confusing-void-expression': 'off',
+        '@typescript-eslint/no-dupe-class-members': 'off',
+        '@typescript-eslint/no-duplicate-imports': 'off',
+        '@typescript-eslint/no-dynamic-delete': 'off',
+        '@typescript-eslint/no-empty-function': 'off',
+        '@typescript-eslint/no-empty-interface': 'off',
+        '@typescript-eslint/no-explicit-any': 'off',
+        '@typescript-eslint/no-extra-non-null-assertion': 'off',
+        '@typescript-eslint/no-extra-parens': 'off',
+        '@typescript-eslint/no-extra-semi': 'off',
+        '@typescript-eslint/no-extraneous-class': 'off',
+        '@typescript-eslint/no-floating-promises': 'off',
+        '@typescript-eslint/no-for-in-array': 'off',
+        '@typescript-eslint/no-implicit-any-catch': 'off',
+        '@typescript-eslint/no-implied-eval': 'off',
+        '@typescript-eslint/no-inferrable-types': 'off',
+        '@typescript-eslint/no-invalid-this': 'off',
+        '@typescript-eslint/no-invalid-void-type': 'off',
+        '@typescript-eslint/no-loop-func': 'off',
+        '@typescript-eslint/no-loss-of-precision': 'off',
+        '@typescript-eslint/no-magic-numbers': 'off',
+        '@typescript-eslint/no-misused-new': 'off',
+        '@typescript-eslint/no-misused-promises': 'off',
+        '@typescript-eslint/no-namespace': 'off',
+        '@typescript-eslint/no-non-null-asserted-optional-chain': 'off',
+        '@typescript-eslint/no-non-null-assertion': 'off',
+        '@typescript-eslint/no-parameter-properties': 'off',
+        '@typescript-eslint/no-redeclare': 'off',
+        '@typescript-eslint/no-require-imports': 'off',
+        '@typescript-eslint/no-shadow': 'off',
+        '@typescript-eslint/no-this-alias': 'off',
+        '@typescript-eslint/no-throw-literal': 'off',
+        '@typescript-eslint/no-type-alias': 'off',
+        '@typescript-eslint/no-unnecessary-boolean-literal-compare': 'off',
+        '@typescript-eslint/no-unnecessary-condition': 'off',
+        '@typescript-eslint/no-unnecessary-qualifier': 'off',
+        '@typescript-eslint/no-unnecessary-type-arguments': 'off',
+        '@typescript-eslint/no-unnecessary-type-assertion': 'off',
+        '@typescript-eslint/no-unnecessary-type-constraint': 'off',
+        '@typescript-eslint/no-unsafe-argument': 'off',
+        '@typescript-eslint/no-unsafe-assignment': 'off',
+        '@typescript-eslint/no-unsafe-call': 'off',
+        '@typescript-eslint/no-unsafe-member-access': 'off',
+        '@typescript-eslint/no-unsafe-return': 'off',
+        '@typescript-eslint/no-unused-expressions': 'off',
+        '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
I don't know if this is caused by this rule:
this line https://github.com/FLOIP/flow-builder/blob/eslint-upgraded-and-fixed/src/components/interaction-designer/flow-editors/ModesEditor.vue#L36 gives us an error

```
ERROR in /Users/rs/Documents/Voto/floip/flow-builder/src/components/interaction-designer/flow-editors/ModesEditor.vue(35,13):
35:13 'availableModes' is declared but its value is never read.
    33 |     @Prop() readonly flow!: IFlow
    34 |
  > 35 |     private availableModes = Object.values(SupportedMode)
       |             ^
    36 |
    37 |     get flowSelectedModes(): SupportedMode[] {
    38 |       return this.flow.supported_modes
```
    
While `availableModes` is used in template, any idea how to fix it?

Reply:
I turned off this rule in tsconfig, so it' should be good now.